packages/app/src/pages/NewSpec.vue

```diff
@@ -14,9 +14,12 @@
       <Button @click="codeGenTypeClicked('integration')">
         Generate Integration
       </Button>
+      <Button @click="codeGenTypeClicked('scaffoldIntegration'); scaffoldIntegration.executeMutation({})">
```

Comment:
I think we should create a method on `<script>` for this, to keep `<template>` more thin and simple.

Reply:
This page was deleted with Jess' recent merge of the `create-spec` flow, but yeah I should have had moved that into a function.